lib/assets/javascripts/cartodb/table/layer_panel_view.js

```diff
@@ -226,6 +228,33 @@
       this.model.save({ 'visible': !this.model.get('visible') });
     },
 
+
+    _addPolygonFilter: function(e) {
+      if( this.dataLayer.get('type') !== 'CartoDB' ) {
+        console.log("Skipping layer because it doesn't have filters");
```

Comment:
Is there a better method to use so it doesn't log in production, or just don't log?